tests/baselines/reference/completionsClassMembers1.baseline

```diff
@@ -1,11 +1,39 @@
+=== /tests/cases/fourslash/completionsClassMembers1.ts ===
+// interface I {
+//     method(): void;
+// }
+// 
+// export class C implements I {
+//     property = "foo"
+// 
+//     
+//     ^
```

Comment:
Nit: lots, but not all, of these have 2 extra blank lines. Is that easy to fix?

Reply:
I think what you're seeing is from the source, not the new formatting:

```ts
export class C implements I {
  property = "foo"

  /*1*/
```

It's easy to fix but hard to know whether I've fixed them all.

Reply:
Got it, I don’t have a problem if it was from the source. I just assumed it was a baseline issue that it made it _look_ like the source was doing weird stuff.